Vincent/Screens/ArtItem/ArtItemCategoryModalViewCell.swift

```diff
@@ -0,0 +1,66 @@
+//
+//  ArtItemCategoryModalViewCell.swift
+//  Vincent
+//
+//  Created by 김연호 on 2022/10/13.
+//
+
+import UIKit
+
+import SnapKit
+import Then
+
+final class ArtItemCategoryModalViewCell: BaseCollectionViewCell {
+
+    let artCategoryModalLabel = UILabel().then {
+        $0.font = .preferredFont(forTextStyle: .headline, weight: .semibold)
+        $0.textAlignment = .center
+        $0.backgroundColor = .clear
+        $0.textColor = .black
+    }
+
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        render()
+        configUI()
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    override func render() {
+        contentView.clipsToBounds = true
+        contentView.layer.borderColor = UIColor.gray.cgColor
+        contentView.layer.borderWidth = 1
+        contentView.layer.cornerRadius = 10
+        contentView.backgroundColor = .clear
+        contentView.addSubview(artCategoryModalLabel)
+    }
+
+    override func configUI() {
+        artCategoryModalLabel.snp.makeConstraints {
+            $0.centerX.equalTo(contentView.snp.centerX)
```

Comment:
p3
```suggestion
            $0.centerX.equalToSuperView()
```
부모뷰와 동일한 x,y 중앙정렬은 위와같이 편하게 간소화할 수 있습니다 참고하세용

Reply:
collectionView에서  ContentView가 부모뷰인가요?!

Reply:
음.. 질문의 의도는 잘모르겠지만 collectionView에는 contentView가 없습니다.
ViewController가 view 프로퍼티를 가지는 것처럼 collectionViewCell은 contentView 라는 프로퍼티를 가지는 거로 알고 있습니다!

Reply:
```swift
contentView.addSubview(artCategoryModalLabel)
```
이거 하셨잖아요.
`artCategoryModalLabel`의 부모뷰가 `contentView`가 되는거죠!